nasa-gibs/worldview
# WV-3238: Added 3 AMSR2 L3 layers

## Description

Fixes #WV-3238 .

Added 3 Daily Level 3 AMSR2 layers
- Total Precipitable Water
- Cloud Liquid Water Path
- Ocean Wind Speed

## How To Test

1. Open with [these URL parameters](http://localhost:3000/?l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,AMSRU2_L3_Ocean_Wind_Speed_Daily,AMSRU2_L3_Total_Precipitable_Water_Daily,AMSRU2_L3_Cloud_Liquid_Water_Daily,MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2024-11-21-T21%3A50%3A44Z)
2. Check to make sure the layers display, you can take an image snapshot, layer descriptions are accurate.

@nasa-gibs/worldview
